ItalianTranslation_PlanetCoaster.xml

```diff
@@ -5766,189 +5766,189 @@
 		<entry key="TrackElementDesc_TopHat_Up" translation="Top Hat Incline Description"/>
 		<entry key="TrackElementDesc_TopHat_UpRoll" translation="Incline Roll Description"/>
 		<entry key="TrackElementDesc_TopHat_UpTwist" translation="Incline Twist Description"/>
-		<entry key="TrackElementName_ChannelEntry" translation="Channel Entry"/>
-		<entry key="TrackElementName_ChannelTrack" translation="Channel Track"/>
+		<entry key="TrackElementName_ChannelEntry" translation="Entrata Canale"/>
```

Comment:
Anche qui sei sicuro che sia "canale" e non per esempio "tracciato"?

Reply:
Mi sono basato sulla stringa: TrackElementDesc_ChannelTrack" translation="Channel of water that pushes the boat along."/> , quindi ho presunto si trattasse di attrazione acquatica. Alcune voci si trovano se provi a creare una montagna russa acquatica (parzialmente tradotte)